docs/fundamentals/aggregation-builder.txt

```diff
@@ -0,0 +1,513 @@
+
+.. _laravel-aggregation-sort-stage-example:
+
+Sort Stage Example
+~~~~~~~~~~~~~~~~~~
+
+You can chain the ``sort()`` method to your aggregation pipeline to specify
+the documents' output order.
+
+You can add this aggregation stage anywhere in the pipeline. It is often
+placed after the group stage since it can depend on the grouped data. We
+recommend placing the sort stage as late as possible in the pipeline to limit
+the data it processes.
+
+To specify an ascending sort, set the field value to the ``Sort::Asc`` enum.
+
+To specify a descending sort, set the field value to the ``Sort::Desc`` enum.
+
+This example shows a ``sort()`` aggregation pipeline stage that sorts the
+documents by the ``name`` field to ``Sort::Desc``, corresponding to reverse
+alphabetical order. Click the :guilabel:`{+code-output-label+}` button to see
+the documents returned by running the code:
+
+.. io-code-block::
+
+   .. input:: /includes/fundamentals/aggregation/AggregationsBuilderTest.php
+      :language: php
+      :dedent:
+      :start-after: begin aggregation sort stage
+      :end-before: end aggregation sort stage
+
+   .. output::
+      :language: json
+      :visible: false
+
+      [
+        {
+          "_id": ...,
+          "name": "Janet Doe",
+          "occupation": "designer",
+          "birthday": {
+            "$date": {
+              "$numberLong": "541728000000"
+            }
+          }
+        },
+        {
+          "_id": ...,
+          "name": "Francois Soma",
+          "occupation": "engineer",
+          "birthday": {
+            "$date": {
+              "$numberLong": "886377600000"
+            }
+          }
+        },
+        {
+          "_id": ...,
+          "name": "Ellis Lee",
+          "occupation": "designer",
+          "birthday": {
+            "$date": {
+              "$numberLong": "834019200000"
+            }
+          }
+        },
+        {
+          "_id": ...,
+          "name": "Eliud Nkosana",
+          "occupation": "engineer",
+          "birthday": {
+            "$date": {
+              "$numberLong": "449884800000"
+            }
+          }
+        },
+        {
+          "_id": ...,
+          "name": "Bran Steafan",
+          "occupation": "engineer",
+          "birthday": {
+            "$date": {
+              "$numberLong": "894326400000"
+            }
+          }
+        },
+        {
+          "_id": ...,
+          "name": "Alda Gröndal",
+          "occupation": "engineer",
+          "birthday": {
+            "$date": {
+              "$numberLong": "1009843200000"
+            }
+          }
+        }
+      ]
+
+.. _laravel-aggregation-project-stage-example:
+
+Project Stage Example
+~~~~~~~~~~~~~~~~~~~~~
+
+You can chain the ``project()`` method to your aggregation pipeline to specify
+which fields from the documents to display by this stage.
+
+To specify fields to include, pass the name of a field and a truthy value,
```

Comment:
Q: could including words like "truthy" and "falsy" confuse readers?

Reply:
I think it's a relatively common term and not obscure jargon. I provided examples in the qualifying phrase immediately after. Let me know if you had something else in mind for clarifying this.